backend/src/schema/resolvers/reports.spec.js

```diff
@@ -147,34 +182,54 @@ describe('report', () => {
               id: 'c34',
               content: 'Robert getting tired.',
             })
-            variables = { id: 'c34' }
+            variables = {
+              id: 'c34',
+            }
           })
 
           it('returns type "Comment"', async () => {
             returnedObject = '{ type }'
             await expect(action()).resolves.toEqual({
-              report: { type: 'Comment' },
+              report: {
+                type: 'Comment',
+              },
             })
           })
 
           it('returns resource in comment attribute', async () => {
             returnedObject = '{ comment { content } }'
             await expect(action()).resolves.toEqual({
-              report: { comment: { content: 'Robert getting tired.' } },
+              report: {
+                comment: {
+                  content: 'Robert getting tired.',
+                },
+              },
             })
           })
         })
 
+        /* An der Stelle würde ich den c34 noch mal prüfen, diesmal muss aber eine error meldung kommen.
+           At this point I would check the c34 again, but this time there must be an error message. */
+
         describe('reported resource is a tag', () => {
           beforeEach(async () => {
-            await factory.create('Tag', { id: 't23' })
-            variables = { id: 't23' }
+            await factory.create('Tag', {
+              id: 't23',
+            })
+            variables = {
+              id: 't23',
+            }
           })
 
           it('returns null', async () => {
-            await expect(action()).resolves.toEqual({ report: null })
+            await expect(action()).resolves.toEqual({
+              report: null,
+            })
           })
         })
+
+        /* An der Stelle würde ich den t23 noch mal prüfen, diesmal muss aber eine error meldung kommen.
```

Comment:
> [<img alt="roschaefer" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/roschaefer) **Authored by [roschaefer](https://github.com/roschaefer)**
_<time datetime="2019-06-21T12:52:18Z" title="Friday, June 21st 2019, 2:52:18 pm +02:00">Jun 21, 2019</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -147,34 +182,54 @@ describe('report', () => {
               id: 'c34',
               content: 'Robert getting tired.',
             })
-            variables = { id: 'c34' }
+            variables = {
+              id: 'c34',
+            }
           })
 
           it('returns type "Comment"', async () => {
             returnedObject = '{ type }'
             await expect(action()).resolves.toEqual({
-              report: { type: 'Comment' },
+              report: {
+                type: 'Comment',
+              },
             })
           })
 
           it('returns resource in comment attribute', async () => {
             returnedObject = '{ comment { content } }'
             await expect(action()).resolves.toEqual({
-              report: { comment: { content: 'Robert getting tired.' } },
+              report: {
+                comment: {
+                  content: 'Robert getting tired.',
+                },
+              },
             })
           })
         })
 
+        /* An der Stelle würde ich den c34 noch mal prüfen, diesmal muss aber eine error meldung kommen.
+           At this point I would check the c34 again, but this time there must be an error message. */
+
         describe('reported resource is a tag', () => {
           beforeEach(async () => {
-            await factory.create('Tag', { id: 't23' })
-            variables = { id: 't23' }
+            await factory.create('Tag', {
+              id: 't23',
+            })
+            variables = {
+              id: 't23',
+            }
           })
 
           it('returns null', async () => {
-            await expect(action()).resolves.toEqual({ report: null })
+            await expect(action()).resolves.toEqual({
+              report: null,
+            })
           })
         })
+
+        /* An der Stelle würde ich den t23 noch mal prüfen, diesmal muss aber eine error meldung kommen.
```


```suggestion
        /* 
```
Well, if you want to add test cases, go ahead :slightly_smiling_face: